app/src/main/java/com/example/minutejournal/AddEntryActivity.kt

```diff
@@ -0,0 +1,31 @@
+package com.example.minutejournal
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.ImageView
+import android.widget.TextView
+
+class AddEntryActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_add_entry)
+
+        val button = findViewById<TextView>(R.id.doneBtn)
```

Comment:
1) Give this component a more descriptive name than "button".
2) Use Kotlin synthetic imports, so that you do not have to invoke findViewById to retrieve the view from the layout.